guide/src/index.html

```diff
@@ -1,7 +1,7 @@
 <html>
     <head>
         <meta charset="utf-8" />
-        <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css' />
+        <link href='https://fonts.googleapis.com/css?family=Roboto:300,400' rel='stylesheet' type='text/css' />
```

Comment:
Not yet, I came across this while integrating with troposphere. I want to prove it before filing an issue with them, as it could be something to do with Roboto on my computer.

The buttons on their docs visually are lightweight but the css is using font-weight of 500. Toggling this off and on has no effect on the button. In our docs the button text is visually heavier weight and when the 500 rule was toggled off the weight looked like MUI's docs.
<img width="163" alt="screen shot 2017-08-21 at 2 04 31 pm" src="https://user-images.githubusercontent.com/7366338/29538247-b7c2f560-8679-11e7-807e-6311ac070624.png">

<img width="154" alt="screen shot 2017-08-21 at 2 04 46 pm" src="https://user-images.githubusercontent.com/7366338/29538253-bc3c09b0-8679-11e7-8370-c555b3b242e7.png">

I'm not using 500 for anything so was just making it look right for now with the intent to fix it later